annotations/annotations_api.go

```diff
@@ -15,10 +15,10 @@ const annotationsEndpoint = "/annotations"
 
 const syntheticContentUUID = "4f2f97ea-b8ec-11e4-b8e6-00144feab7de"
 
-type AnnotationsAPI interface {
+type API interface {
```

Comment:
You are right. I though about it. I was actually thinking to move this to a different package, since is something that belongs to UPP, but I was scared to increase the size of this PR by doing this movement. 
There are two options in my opinion:
- rename it as `UPPAnnotationsAPI` - in the `main` package, it should look `annotations.UPPAnnotationsAPI`
- Move it in a `upp` package and restore the original name - in the `main` package, it should look `upp.AnnotationsAPI`

I think that `ReadAPI` creates confusion with the `RW`

Reply:
Of those two I would go for `UPPAnnotationsAPI` but I don't have strong views on it.

Reply:
Done